src/bpf_assembler.h

```diff
@@ -0,0 +1,168 @@
+// BPF macro assembler copied from bcc/libbpf.h
```

Comment:
what is this file for?

Reply:
For the test programs in bpffeature. We use assembler there